### ISSUE TITLE
OpenTSDB complains in Bosun UI about parsing JSON

### ISSUE THREAD

**Issue by thenewwazoo** [bosun-monitor/bosun#1568]
Currently running a build of 4b38062, and running query results in:
```
opentsdb: {"start":"1h-ago","queries":[{"aggregator":"sum","metric":"linux.loadavg_1_min","rateOptions":{},"tags":{"host":"ops-opentsdb-0|ops-opentsdb-1|ops-opentsdb-2|ops-opentsdb-3|ops-opentsdb-4|ops-opentsdb-5|ops-opentsdb-6|ops-opentsdb-7"},"filters":[{"type":"iwildcard","tagk":"host","filter":"ops-o*","groupBy":true}]}]}: Unable to parse the given JSON
```

appearing in the UI. Bosun has config file line `tsdbVersion = 2.1`, and per kylebrandt's instruction I've validated that `$scope.filterSupport` is not defined.

**Comment by thenewwazoo** [comment]
This is the GET request that emits the error:
```
GET http://ops-bosun-0.example.com:8070/api/graph?b64=eyJzdGFydCI6IjFoLWFnbyIsInF1ZXJpZXMiOlt7ImFnZ3JlZ2F0b3IiOiJzdW0iLCJtZXRyaWMiOiJsaW51eC5sb2FkYXZnXzFfbWluIiwicmF0ZSI6ZmFsc2UsInJhdGVPcHRpb25zIjp7fSwiZGVyaXZhdGl2ZSI6ImF1dG8iLCJkcyI6IiIsImRzdGltZSI6IiIsInRhZ3MiOnsiaG9zdCI6Im9wcy1vKiJ9LCJnYkZpbHRlcnMiOnsiaG9zdCI6eyJ0eXBlIjoiYXV0byIsInRhZ2siOiJob3N0IiwiZmlsdGVyIjoib3BzLW8qIiwiZ3JvdXBCeSI6dHJ1ZX19LCJuR2JGaWx0ZXJzIjp7fSwiZmlsdGVycyI6W3sidHlwZSI6Iml3aWxkY2FyZCIsInRhZ2siOiJob3N0IiwiZmlsdGVyIjoib3BzLW8qIiwiZ3JvdXBCeSI6dHJ1ZX1dLCJkb3duc2FtcGxlIjoiIn1dfQ%3D%3D&autods=1140&autorate=0```

**Comment by thenewwazoo** [comment]
Confirmed fixed in 68af647
